config/initializers/allowed_domains.rb

```diff
@@ -1,6 +1,7 @@
 ALLOWED_DOMAINS = [
-  'cabinet-office.gov.uk',
-  'digital.cabinet-office.gov.uk'
+  'cabinetoffice.gov.uk',
+  'digital.cabinet-office.gov.uk',
+  'yaml.farm'
```

Comment:
I'll rip this out, since it doesn't need to be here.

I'll try to declare this list of domains as a environment variable for flexibility.

That was my plan all along, but only when I created the PR did I realise...